### PR TITLE
v0.0.17

### PR DESCRIPTION
**Pull request by aappling-usgs**
Two small changes plus a version bump:

1) You can now call `scipiper:::get_dependency_status('targetname')` without writing all the code to get a remake_object. This is still an internal function but has been proving its worth for debugging recently.

2) In task plans, objects with complex names (e.g., 'nhdhr_23094-019867') were giving trouble in the combiner call, where remake failed to parse them. This PR adds tick quotes (\`) around objects in the combiner calls by default. You might want to skip this change for current built task plans, in which case you can specify `tickquote_combinee_objects = FALSE` in the call to `create_task_makefile` so that your task plan stays unchanged. If you do this, you'll get a warning about this argument being deprecated (already, even though this PR introduces it) because in the long run I think we'll be better off with always tickquoting.

### CODE REVIEW

File: R/create_task_makefile.R [hunk 1]
```diff
@@ -130,7 +136,7 @@ create_task_makefile <- function(
     if (('remake' %:::% 'target_is_file')(target, file_extensions = pos_extensions)){
       paste0("'", target, "'")
     } else {
-      target
+      if(tickquote_combinee_objects) paste0("`", target, "`") else target
```
Comment (jread-usgs) [inline R/create_task_makefile.R]: well that was pretty easy/clean

Comment (aappling-usgs) [reply R/create_task_makefile.R]: yeah, i should have looked before worrying/complaining =)

Comment (jread-usgs) [reply R/create_task_makefile.R]: did you want `tickquote_combinee_objects` or `tickquote_combine[r]_objects`?

Comment (aappling-usgs) [reply R/create_task_makefile.R]: i intended `combinee` because it's referring to objects that get combined. What's your thinking?

Comment (jread-usgs) [reply R/create_task_makefile.R]: haha, sounds good. I thought maybe it was a typo, but I get it now.

**Comment by aappling-usgs** [comment]
@jread-usgs Since you've already looked, do you mind reviewing/approving/merging this PR?

**Pull request merged**